package-system/PhysX/FindPhysX.cmake.template

```diff
@@ -55,17 +55,38 @@ set(extra_static_libs ${EXTRA_STATIC_LIBS_NON_MONOLITHIC})
 set(extra_shared_libs ${EXTRA_SHARED_LIBS})
 
 if(LY_MONOLITHIC_GAME)
-    list(APPEND $${MY_NAME}_LIBRARIES
-        $${PATH_TO_STATIC_LIBS}/$${CMAKE_STATIC_LIBRARY_PREFIX}PhysX_static_64$${CMAKE_STATIC_LIBRARY_SUFFIX}
-        $${PATH_TO_STATIC_LIBS}/$${CMAKE_STATIC_LIBRARY_PREFIX}PhysXCooking_static_64$${CMAKE_STATIC_LIBRARY_SUFFIX}
-        $${PATH_TO_STATIC_LIBS}/$${CMAKE_STATIC_LIBRARY_PREFIX}PhysXFoundation_static_64$${CMAKE_STATIC_LIBRARY_SUFFIX}
-        $${PATH_TO_STATIC_LIBS}/$${CMAKE_STATIC_LIBRARY_PREFIX}PhysXCommon_static_64$${CMAKE_STATIC_LIBRARY_SUFFIX}
+    set(MONO_PATH_TO_STATIC_LIBS $${CMAKE_CURRENT_LIST_DIR}/PhysX/static/lib)
+    # The order of PhysX 4.x static libraries is important for monolithic targets.
+    set(IMPORTED_PHYSICS_LIBS
+        PhysX_static_64
+        PhysXPvdSDK_static_64
+        PhysXVehicle_static_64
+        PhysXCharacterKinematic_static_64
+        PhysXExtensions_static_64
+        PhysXCooking_static_64
+        PhysXCommon_static_64
+        PhysXFoundation_static_64
     )
-    if(extra_shared_libs)
-        set($${MY_NAME}_RUNTIME_DEPENDENCIES
-            $${extra_shared_libs}
+    foreach(PHYSICS_LIB $${IMPORTED_PHYSICS_LIBS})
+        add_library($${PHYSICS_LIB}::imported STATIC IMPORTED GLOBAL)
+        set($${PHYSICS_LIB}_PATH $${MONO_PATH_TO_STATIC_LIBS}/$${CMAKE_STATIC_LIBRARY_PREFIX}$${PHYSICS_LIB}$${CMAKE_STATIC_LIBRARY_SUFFIX})
+        set_target_properties($${PHYSICS_LIB}::imported
+            PROPERTIES
+                IMPORTED_LOCATION_DEBUG   $${$${PHYSICS_LIB}_PATH}
+                IMPORTED_LOCATION_PROFILE $${$${PHYSICS_LIB}_PATH}
+                IMPORTED_LOCATION_RELEASE $${$${PHYSICS_LIB}_PATH}
```

Comment:
Since we are only using the release library on for PhysX linux, only a single IMPORTED_LOCATION is needed
```suggestion
                IMPORTED_LOCATION   $${$${PHYSICS_LIB}_PATH}
```